packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -302,14 +303,17 @@ export class WalletConnectConnection implements WalletConnection {
         challenge: string,
         credentialStatements: CredentialStatements
     ): Promise<VerifiablePresentation> {
-        return this.connector.client.request<VerifiablePresentation>({
+        const serializedParams = JSONBigInt.stringify({ challenge, credentialStatements });
+        const params = { paramsJson: serializedParams };
```

Comment:
+1 for naming it with `Json` prefix and if you named `serializedParams` as `paramsJson` it would be a little more concise :)

Reply:
I have renamed it to `paramsJson`.